modules/caddyhttp/staticresp.go

```diff
@@ -71,16 +70,15 @@ Access/request logging and more verbose debug logging can also be enabled.
 
 Response headers may be added using the --header flag for each header field.
 `,
-		Flags: func() *flag.FlagSet {
-			fs := flag.NewFlagSet("respond", flag.ExitOnError)
-			fs.String("listen", ":0", "The address to which to bind the listener")
-			fs.Int("status", http.StatusOK, "The response status code")
-			fs.String("body", "", "The body of the HTTP response")
-			fs.Bool("access-log", false, "Enable the access log")
-			fs.Bool("debug", false, "Enable more verbose debug-level logging")
-			fs.Var(&respondCmdHeaders, "header", "Set a header on the response (format: \"Field: value\"")
-			return fs
-		}(),
+		CobraFunc: func(cmd *cobra.Command) {
+			cmd.Flags().StringP("listen", "l", ":0", "The address to which to bind the listener")
+			cmd.Flags().IntP("status", "s", http.StatusOK, "The response status code")
+			cmd.Flags().StringP("body", "b", "", "The body of the HTTP response")
```

Comment:
```suggestion
			cmd.Flags().StringP("body", "b", "", "The body of the HTTP response")
```

I'm torn on this one, because it almost makes sense to have some parity with `curl`, like `-d` for body, for example. `-b` feels like 'binary'. Hmm.

Reply:
`-d` makes no sense to me. Is that supposed to be "data"? Should we rename the flag in that case? In this case, I think the body is core to this command's usage, so having an easy way to set it makes a lot of sense.

Reply:
Yeah, data.

I agree it should have a short flag. -b is fine, but -d is what comes to mind first for me, since `curl` (the de-facto command for HTTP requests) uses -d.

Reply:
I just want to mention `curl`'s long flag for `-d` is `--data` (along with other variants of that option following `--data-*` like `--data-urlencode` and `--data-raw`)